rrfs-test/scripts/templates/run_mpasjedi_template.sh

```diff
@@ -18,6 +18,8 @@ if [[ $hostname == "h" ]]; then
   platform="hera"
 elif [[ $hostname == "o" ]]; then
   platform="orion"
+elif [[ $hostname == "f" ]]; then
+  platform="jet"
```

Comment:
FYI, at the computing node, the hostname does not necessarily return strings leading with "f", "h" etc.

Reply:
What do you suggest this should be?

Reply:
At this moment, we may use a similar method as what is used by global workflow and rrfs-workflow, i.e. to check specific folders. Here is an example:
https://github.com/NOAA-EMC/rrfs-workflow/blob/dev-sci/ush/fix_rrfs_locations.sh

Hercules and Orion share the same file system but I recalled that `hostname` at their computing nodes returns distinguishable strings between them.

Reply:
ok, we already have a powerful script in RDASApp:
`RDASApp/ush/detect_machine.sh`

Reply:
Yep, I just found that and was going to use it. Thank you!

Reply:
That makes a lot of these scripts a lot simpler. Thanks for the suggestion.

Reply:
@delippi Have you completed the test on all three machines?

Reply:
Give me a moment and I will confirm on all three.

Reply:
@ShunLiu-NOAA, everything still works on Orion and Hera. I had a strange failure on Jet that I need to figure out.

Reply:
@ShunLiu-NOAA, it seems to be working now on Jet as well. I ***think*** the issue was one of Jet's files systems (lsf4) was having issues and something in module load was trying to reach something staged there.

Reply:
@delippi  You are right. /lfs4 had performance issues in the past few days and Admin took Jet down to fix it. Fortunately, it was successfully fixed yesterday.